sources/corepackUtils.ts

```diff
@@ -200,7 +204,7 @@ export async function installVersion(installTarget: string, locator: Locator, {s
 
   stream.pipe(sendTo);
 
-  const algo = build[0] ?? `sha256`;
+  const algo = build[0] ?? `sha512`;
```

Comment:
Why switching to sha512?

Reply:
Because that’s what npm registry for the signature: they sign a combination of the package name & version and the SHA512 of the package archive.